City-of-Helsinki/hitas
# backend: Update initial.json

# Hitas Pull Request

# Description

- Fix `initial.json` being broken after migrations
- Add index data to `initial.json`. These contain at least partially generated values.

## Pull request checklist

Check the boxes for each DoD items that has been completed:

- **Frontend**
- **Backend**
    - [ ] Automatic tests has been added
    - [ ] Database migrations will work in test environment

## Test plan

- Run `./manage.py loaddata initial.json` and verify it works 😛 
- Check that you have indicies loaded

## Tickets

This pull request resolves all or part of the following ticket(s): -
